clar2wasm/tests/wasm-generation/blockinfo.rs

```diff
@@ -122,4 +123,30 @@ proptest! {
         crosscheck_for(StacksEpochId::Epoch30, expected_block.clone(), "stacks-block-height");
         crosscheck_for(StacksEpochId::Epoch24, expected_block, "block-height");
     }
+
+    #[test]
+    fn crosscheck_at_block_sender_no_leak(
+        buf in buffer(32)
+    ) {
+        let expected = Value::okay(Value::Principal(PrincipalData::Standard(StandardPrincipalData::transient())));
+
+        crosscheck(
+            &format!("(begin (at-block {buf} 42) (ok tx-sender))"),
```

Comment:
Why would you check for `tx-sender`? Does it change when you execute an `at-block`?

Reply:
Test for checking at-block scope done in function `crosscheck_at_block_no_leak`.

Reply:
my bad, missed that